backend/services/syson-services/src/main/java/org/eclipse/syson/services/UtilService.java

```diff
@@ -761,4 +767,38 @@ private boolean isPart(Element element) {
     private boolean isAction(Element element) {
         return element instanceof ActionUsage || element instanceof ActionDefinition;
     }
+
+    /**
+     * Return an instance of the actual {@link PartDefinition} associated to the given {@link PartUsage}.
+     *
+     * @param partUsage
+     *         an instance of PartUsage
+     * @return an actual instance of the part definition associated to the given part usage.
+     */
+    public Definition createRightPartDefinitionFrom(PartUsage partUsage) {
+        return (Definition) SysmlFactory.eINSTANCE.create(this.getRightPartDefinitionEClassFrom(partUsage));
+    }
+
+    /**
+     * Return the PartDefinition {@link EClass} corresponding to the given PartUsage.
+     * @param partUsage a PartUsage instance
```

Comment:
Please explain here that the PartUsage parameter could be one of its substype.
That's the whole purpose of this method.

Reply:
Done